build.gradle

```diff
@@ -93,6 +94,8 @@ dependencies {
 	testImplementation project(":workspace-manager-client")
 
 	annotationProcessor group: "org.springframework.boot", name: "spring-boot-configuration-processor"
+	// Used only for logback configuration file.
+	runtime group: 'org.codehaus.groovy', name: 'groovy', version: '2.5.7'
```

Comment:
This should probably go with the normal deps, so above postgres. It's icky to me that we have to bring in something this big just to parse a config, but I don't really see an alternative, since we really don't like xml.